.vscode/settings.json

```diff
@@ -6,10 +6,17 @@
     "files.trimFinalNewlines": true,
     "files.trimTrailingWhitespace": true,
     "files.eol": "\n",
+    "cSpell.language": "en-GB",
     "editor.rulers": [
         120
     ],
+    "protoc": {
+        "options": [
+            "-I${workspaceFolder}/nuclear/message/proto"
+        ]
+    },
     "python.formatting.provider": "black",
+    "yaml.format.printWidth": 120,
```

Comment:
Why are these in here?

Reply:
Because they should be! And I don't want to do another PR to do it :P

Reply:
whats the protoc stuff?

Reply:
Those are so that the compilation/errors for protobuf works properly in vscode. To do that they need the include path